commit 35ef462ec5d18431b3508837c6cc1dd21bc1d73f
Author: Andreas Streichardt <andreas@mop.koeln>
Date:   Thu Apr 21 18:46:01 2022 +0200

    🐛 use chromium, disable sound module (tries to fetch firmware)

diff --git a/ansible/getraenkekasse/files/sound-blacklist.conf b/ansible/getraenkekasse/files/sound-blacklist.conf
new file mode 100644
index 0000000..572ac54
--- /dev/null
+++ b/ansible/getraenkekasse/files/sound-blacklist.conf
@@ -0,0 +1 @@
+install snd_intel_sst_acpi /bin/false
\ No newline at end of file
diff --git a/ansible/getraenkekasse/tasks/main.yml b/ansible/getraenkekasse/tasks/main.yml
index e31f557..3211e97 100644
--- a/ansible/getraenkekasse/tasks/main.yml
+++ b/ansible/getraenkekasse/tasks/main.yml
@@ -25,6 +25,12 @@
         dest: /etc/modprobe.d/nfc-blacklist.conf
         owner: root
         group: root
+    - name: blacklist kernel sound module (needs firmware, spams log)
+      copy:
+        src: files/sound-blacklist.conf
+        dest: /etc/modprobe.d/sound-blacklist.conf
+        owner: root
+        group: root
     - name: copy stornoschluessel udev rules
       copy:
         src: files/52-storno.rules
diff --git a/ansible/kiosk/tasks/main.yml b/ansible/kiosk/tasks/main.yml
index 03bf306..3a613fc 100644
--- a/ansible/kiosk/tasks/main.yml
+++ b/ansible/kiosk/tasks/main.yml
@@ -7,10 +7,14 @@
     - name: install tablet kiosk required packages
       apt:
         pkg:
-          - firefox-esr
+          - chromium
           - phoc
           - foot
           - curl
+    - name: remove pulseaudio
+      apt:
+        name: pulseaudio
+        state: absent
     - name: copy getränkekiosk
       copy:
         src: files/getraenke-kiosk.service
diff --git a/ansible/kiosk/templates/kiosk.sh.j2 b/ansible/kiosk/templates/kiosk.sh.j2
index da5da09..5b5578b 100644
--- a/ansible/kiosk/templates/kiosk.sh.j2
+++ b/ansible/kiosk/templates/kiosk.sh.j2
@@ -1,19 +1,22 @@
 #!/bin/sh
 
-squeekboard &
-
 FAIL="1"
 while [ "$FAIL" -ne "0" ]; do
     sleep 1
-    curl http://{{ public_hostname }} -s -f -L -o /dev/null
+    curl http://getraenkekasse.kalk.space -s -f -L -o /dev/null
     FAIL=$?
 done
 # wait for mete as well so the initial screen shows drinks right away
 FAIL="1"
 while [ "$FAIL" -ne "0" ]; do
     sleep 1
-    curl http://{{ public_hostname }}/mete/api/v1/drinks.json -s -f -L -o /dev/null
+    curl http://getraenkekasse.kalk.space/mete/api/v1/drinks.json -s -f -L -o /dev/null
     FAIL=$?
 done
 
-firefox --kiosk http://{{ public_hostname }}
\ No newline at end of file
+chromium \
+    --enable-features=UseOzonePlatform \
+    --ozone-platform=wayland \
+    --disable-pinch --start-fullscreen \
+    --overscroll-history-navigation=0 \
+    --kiosk http://getraenkekasse.kalk.space
\ No newline at end of file
